internal/request/graphql/schema/collection.go

```diff
@@ -13,6 +13,7 @@ package schema
 import (
```

Comment:
todo: There appears to be a lot of schema/collection description validation taking place in this file.  Please remove as much as you can and instead place it in `internal/db/definition_validation.go`.

This will allow all our schema validation to live in once place, and lets it operates on our lowest-level/core types, instead of within query-language dependent code-spaces.

Reply:
are you referring to the new code in this PR or in general for this file?

Reply:
The new code in this PR.  We have existing validation in this file, but *only* for GQL/SDL specific validation.  The current PR contains a lot of general validation, such as validating that target is an array-float kind - this needs to move.

Reply:
Moved to the collection/schema validation part of the repo.